save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestCreationComponent.kt

```diff
@@ -234,7 +217,9 @@ private fun contestCreationComponent() = FC<ContestCreationComponentProps> { pro
                         inputTextFormRequired {
                             form = InputTypes.CONTEST_TEST_SUITE_IDS
                             conflictMessage = null
-                            textValue = testSuites.map { it.name }.sorted().joinToString(", ")
+                            textValue = contestDto.testSuites.map { it.name }
```

Comment:
maybe add version here? for example `${it.name} (${it.version})`

Reply:
To my mind, the line will be TOO long to be readable